backend/Dockerfile

```
@@ -0,0 +1,32 @@
+FROM alpine:latest AS runtime
+
+# Install ca-certificates
+RUN apk --no-cache add ca-certificates
+
+WORKDIR /app
+
+# Copy the compiled binary from builder
+COPY --from=builder /app/main .
+
+# Health check endpoint
+HEALTHCHECK --interval=30s --timeout=3s --start-period=5s --retries=3 CMD ["/app/main", "health"] || exit 1
```

Hadolint (2.14.0)

[error] unexpected '|'
expecting '\', a new line followed by the next instruction, or at least one space

(DL1000)